### DISCUSSION TITLE
System-wide Pure Black with warm color font or warm everything else.

### DISCUSSION THREAD

**Discussion by wezfaas** [Mahmud0808/ColorBlendr#138] [Ideas]
Thanks to the background lightness function, I was able to create pure black background on some UI.  However, I've been using "night light" (Pixel phone) for years now and can do without, but that kind of negate the pure black background.

Is it possible to change the font color system wide?  I tried "tint text color", but I cannot see any difference.
What about replacing a certain color palette to display a different color?

It seems the Pitch black theme in dark mode only affects ColorBlendr itself?

I definitely think a user guide on explaining what the functions actually do would be good, at least to keep user expectations in check.

---
"night light" gives your display a reddish-yellow hue and blocks a significant amount of the emitted blue wavelengths.

**Comment by Mahmud0808** [comment]
Yes, you can change certain colors to display a different color. From ColorBlendr settings enable `Override colors manually`, after that go to `Color palette` page and change whichever color you want from there.

Pitch black theme affects all apps which are using material 3 colors. If not, just select the app in `Force per app theme`. Currently it might be broken on android 15 settings homepage.

If you want reddish yellow hue with pitch black, you can just replace the colors marked 800 and 900 of last 2 palette in `Color palette` page with yellowish black color. That should do the job.

**Reply by wezfaas** [reply]
Thanks @Mahmud0808,

Because of the override colors function, I think a good feature would be able to determine what color palettes other app is using.  My silly manual way would be screenshot the app, open the image in Colorblendr and determine the color palette value on click point, or even open a dialog to let user to choose the overriding color.

Is there any easy method to find out if the app uses material color?  I tested the force per app theme, and find that the apps I tested Firefox and Currency fx, their check ticks do not stick.  ie. I ticked it, exit the list, go back in, it's un-ticked.  Does it mean they are not using material3?

The one that does stick, Lawnchair, it doesn't seem to be working.  I did restart the app through its internal menu...

I think this should be the correct document for those who are also interested in material color: https://m2.material.io/design/color/the-color-system.html#color-usage-and-palettes

**Reply by Mahmud0808** [reply]
> Because of the override colors function, I think a good feature would be able to determine what color palettes other app is using.

This is not possible as not every apps use same color. Google themselves use diffferent color on their different apps instead of sticking to a specific color for every apps.

> My silly manual way would be screenshot the app, open the image in Colorblendr and determine the color palette value on click point, or even open a dialog to let user to choose the overriding color.

Use [Pixolor](https://play.google.com/store/apps/details?id=com.embermitre.pixolor.app&hl=en) app. This is what I use most of the time to find specific color code and compare it in ColorBlendr.

> Is there any easy method to find out if the app uses material color?

If the app changes color when you change wallpaper, or it matches your wallpaper's color then yes, it uses Material You colors.

> I ticked it, exit the list, go back in, it's un-ticked. Does it mean they are not using material3?

If it unticks itself then it cannot be themed.

> https://m2.material.io/design/color/the-color-system.html#color-usage-and-palettes

This is for material 2 design (you can see the link starting with m2), google now uses material 3 design.
